diff --git a/node_modules/@cypress/vite-dev-server/dist/makeCypressPlugin.d.ts b/node_modules/@cypress/vite-dev-server/dist/makeCypressPlugin.d.ts
index c749bf7..2bbf811 100644
--- a/node_modules/@cypress/vite-dev-server/dist/makeCypressPlugin.d.ts
+++ b/node_modules/@cypress/vite-dev-server/dist/makeCypressPlugin.d.ts
@@ -1,4 +1,4 @@
 /// <reference types="node" />
 import { EventEmitter } from 'events';
 import { Plugin } from 'vite';
-export declare const makeCypressPlugin: (projectRoot: string, supportFilePath: string, devServerEvents: EventEmitter) => Plugin;
+export declare const makeCypressPlugin: (projectRoot: string, supportFilePath: string, publicPath: string, devServerEvents: EventEmitter) => Plugin;
diff --git a/node_modules/@cypress/vite-dev-server/dist/makeCypressPlugin.js b/node_modules/@cypress/vite-dev-server/dist/makeCypressPlugin.js
index db70126..6907efb 100644
--- a/node_modules/@cypress/vite-dev-server/dist/makeCypressPlugin.js
+++ b/node_modules/@cypress/vite-dev-server/dist/makeCypressPlugin.js
@@ -6,18 +6,18 @@ const mustache_1 = require("mustache");
 const pluginName = 'cypress-transform-html';
 const indexHtmlPath = path_1.resolve(__dirname, '../index-template.html');
 const readIndexHtml = () => fs_1.readFileSync(indexHtmlPath).toString();
-function handleIndex(indexHtml, projectRoot, supportFilePath, req, res) {
-    const specPath = `/${req.headers.__cypress_spec_path}`;
+function handleIndex(indexHtml, projectRoot, publicPath, supportFilePath, req, res) {
+    const specPath = `${publicPath}/${req.headers.__cypress_spec_path}`;
     const supportPath = supportFilePath ? `/${path_1.relative(projectRoot, supportFilePath)}` : null;
     res.end(mustache_1.render(indexHtml, { supportPath, specPath }));
 }
-exports.makeCypressPlugin = (projectRoot, supportFilePath, devServerEvents) => {
+exports.makeCypressPlugin = (projectRoot, supportFilePath, publicPath, devServerEvents) => {
     return {
         name: pluginName,
         enforce: 'pre',
         configureServer: (server) => {
             const indexHtml = readIndexHtml();
-            server.middlewares.use('/index.html', (req, res) => handleIndex(indexHtml, projectRoot, supportFilePath, req, res));
+            server.middlewares.use(`${publicPath}/index.html`, (req, res) => handleIndex(indexHtml, projectRoot, publicPath, supportFilePath, req, res));
         },
         handleHotUpdate: () => {
             devServerEvents.emit('dev-server:compile:success');
diff --git a/node_modules/@cypress/vite-dev-server/dist/startServer.js b/node_modules/@cypress/vite-dev-server/dist/startServer.js
index e5cd080..b762af2 100644
--- a/node_modules/@cypress/vite-dev-server/dist/startServer.js
+++ b/node_modules/@cypress/vite-dev-server/dist/startServer.js
@@ -18,18 +18,18 @@ const path_1 = require("path");
 const makeCypressPlugin_1 = require("./makeCypressPlugin");
 const debug = debug_1.default('cypress:vite-dev-server:start');
 // TODO: Pull in types for Options so we can infer these
-const serverConfig = (projectRoot, supportFilePath, devServerEvents) => {
+const serverConfig = (projectRoot, supportFilePath, publicPath, devServerEvents) => {
     return {
         root: path_1.resolve(__dirname, projectRoot),
         base: '/__cypress/src/',
-        plugins: [makeCypressPlugin_1.makeCypressPlugin(projectRoot, supportFilePath, devServerEvents)],
+        plugins: [makeCypressPlugin_1.makeCypressPlugin(projectRoot, supportFilePath, publicPath, devServerEvents)],
         server: {
             port: 0,
         },
     };
 };
 const resolveServerConfig = ({ viteConfig, options }) => {
-    const defaultServerConfig = serverConfig(options.config.projectRoot, options.config.supportFile, options.devServerEvents);
+    const defaultServerConfig = serverConfig(options.config.projectRoot, options.config.supportFile, options.config.devServerPublicPathRoute, options.devServerEvents);
     const requiredOptions = {
         base: defaultServerConfig.base,
         root: defaultServerConfig.root,
