diff --git a/node_modules/vite/dist/node/chunks/dep-efe32886.js b/node_modules/vite/dist/node/chunks/dep-efe32886.js
index 0179208..2778b7d 100644
--- a/node_modules/vite/dist/node/chunks/dep-efe32886.js
+++ b/node_modules/vite/dist/node/chunks/dep-efe32886.js
@@ -59337,6 +59337,7 @@ async function transformRequest(url, { config, pluginContainer, moduleGraph, wat
     }
 }
 
+const NEW_DEPENDENCY_BUILD_TIMEOUT = 100
 const debugCache = createDebugger('vite:cache');
 const isDebug = !!process.env.DEBUG;
 const knownIgnoreList = new Set(['/', '/favicon.ico']);
@@ -59352,8 +59353,20 @@ function transformMiddleware(server) {
             !((_a = req.url) === null || _a === void 0 ? void 0 : _a.startsWith(CLIENT_PUBLIC_PATH)) &&
             !((_b = req.url) === null || _b === void 0 ? void 0 : _b.includes('vite/dist/client'))) {
             // missing dep pending reload, hold request until reload happens
-            server._pendingReload.then(() => res.end());
-            return;
+            server._pendingReload.then(() =>
+              // If the refresh has not happened after timeout, Vite considers
+              // something unexpected has happened. In this case, Vite
+              // returns an empty response that will error.
+              setTimeout(() => {
+                // status code request timeout
+                res.statusCode = 408
+                res.end(
+                  `<h1>[vite] Something unexpected happened while optimizing "${req.url}"<h1>` +
+                  `<p>The current page should have reloaded by now</p>`
+                )
+              }, NEW_DEPENDENCY_BUILD_TIMEOUT)
+            )
+          return;
         }
         let url;
         try {
